bsc-plugin/src/plugin.spec.ts

```diff
@@ -2143,6 +2117,85 @@ describe('RooibosPlugin', () => {
             expect(findMethod('getAllTestSuitesNames').func.body.statements).to.be.empty;
             expect(findMethod('getIgnoredTestInfo').func.body.statements).to.be.empty;
         });
+
+        const sep = '\n                                    ';
+        const params: [string[], string][] = [
+            [[], 'rooibos_ConsoleTestReporter'],
+            [['CONSOLE'], 'rooibos_ConsoleTestReporter'],
+            [['MyCustomReporter'], 'MyCustomReporter'],
+            [['JUnit', 'MyCustomReporter'], `rooibos_JUnitTestReporter${sep}MyCustomReporter`]
+        ];
+        it('adds custom test reporters', async () => {
+            for (const [reporters, expected] of params) {
```

Comment:
I was looking for a way to parameterise this test, but I'm not sure if there's a standard convention (I'm not really used to mocha). Let me know if there's a better way to do this and I'll change it.